### PR TITLE
adds missing currencies to show inside currency1 menu

### PR DESCRIPTION
**Pull request by kaincenteno**
Adds missing currencies inside Currency1 in both packet and sql db as of 10NOV2020
Also added migration script for existing servers to add those columns

Didn't add daily tally to the import as it already exists as a different file, but not for the sql import script or found in the currency1.cpp packet

This should help new members freaking out about having random numbers on their currencies. I will work on currency2 on an upcoming PR.

**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

File: tools/migrations/currency_columns.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+		ADD COLUMN `rems_ch7` tinyint(3) unsigned NOT NULL DEFAULT 0, \
+		ADD COLUMN `rems_ch8` tinyint(3) unsigned NOT NULL DEFAULT 0, \
+		ADD COLUMN `rems_ch9` tinyint(3) unsigned NOT NULL DEFAULT 0, \
+		ADD COLUMN `rems_ch10` tinyint(3) unsigned NOT NULL DEFAULT 0, \
+		ADD COLUMN `reclamation_marks` smallint(3) unsigned NOT NULL DEFAULT 0, \
+		ADD COLUMN `unity_accolades` int(5) unsigned NOT NULL DEFAULT 0, \
+		ADD COLUMN `deeds` smallint(5) unsigned NOT NULL DEFAULT 0;")
+		db.commit()
+	except mysql.connector.Error as err:
+		print("Something went wrong: {}".format(err))
```
Comment (zach2good) [inline tools/migrations/currency_columns.py]: Newline at the end of this file please

File: sql/char_points.sql [hunk 2]
```diff
@@ -54,6 +54,7 @@ CREATE TABLE `char_points` (
   `chocobuck_sandoria` smallint(4) unsigned NOT NULL DEFAULT '0',
   `chocobuck_bastok` smallint(4) unsigned NOT NULL DEFAULT '0',
   `chocobuck_windurst` smallint(4) unsigned NOT NULL DEFAULT '0',
+  `daily_tally` int(5) signed NOT NULL DEFAULT '-1'
```
Comment (zach2good) [inline sql/char_points.sql]: Missing a `,` at the end of this line

**Review by zach2good — CHANGES_REQUESTED** [review]
All looks reasonable enough, will be ready to go once that missing comma and style issue are fixed 👍

**Comment by kaincenteno** [comment]
Changed sone @zach2good  =D

**Review by zach2good — APPROVED** [review]

**Pull request merged**